invenio_rdm_records/cli.py

```diff
@@ -39,6 +42,86 @@ def demo():
     click.secho('Created records!', fg='green')
 
 
+@rdm_records.command('demo_submission_requests')
+@click.option("-u", "--user", type=click.STRING)
+@with_appcontext
+def demo_submission_requests(user=system_identity.id):
```

Comment:
Open question: should this be moved above in the `demo` func? Otherwise, we will have to document that this extra step should also be done.